spec.bs

```diff
@@ -614,6 +705,55 @@ minimum delay to deliver an [=aggregatable report=].
 random delay to deliver an [=aggregatable report=]. This delay is additional to
 the [=minimum report delay=].
 
+<dfn>All budgeting scopes</dfn> is a non-empty [=list=] of [=budgeting scopes=].
+
+<div class="advisement"><span class="marker">WARNING:</span> When defining a
+[=budgeting scope=], be aware that certain values of [=budgeting scope/max
+budget=] are incompatible with the way that
+{{PrivateAggregation/reserveNamedBudgets}} interprets budget fractions.
+Implementers must verify that every slice <math><mi>n</mi></math> of their
```

Comment:
Wondering if we just declare that only powers of two are ok? E.g. along the lines of "Implementers must use a max budget of the form 2^k with k <= 52. Note: This ensures that i/|maxBudget| can be exactly represented with a `Number` for all integral i in [0,|maxBudget|]."

Alternatively, I think we could probably support any value by incrementing the fraction up to the next valid double (i.e. by adding Math.floor(Math.log2(fraction)) * Number.EPSILON), but that's probably too hacky to be worth exploring if we can just require powers of two.

(Side note: do you know why pr-preview doesn't seem to be working? Would be nice to see all the rendered equations!)